src/CSSBattle/8_ForkingCrazy/Tai.html

```diff
@@ -0,0 +1,58 @@
+<div class="base"></div>
+<div class="middle"></div>
+<div class="prong"></div>
+<div class="prong" id="cut"></div>
+
+<style>
+  body {
+    width: 400px;
+    height: 300px;
+    background: #6592cf;
+    display: flex;
+    justify-content: center;
+    margin: 0;
+  }
+  div {
+    background: #060f55;
+    position: absolute;
+  }
+  .base {
+    width: 20px;
+    height: 80px;
+    bottom: 0px;
+  }
+  .middle {
+    width: 140px;
+    height: 100px;
+    background: #060f55;
+    position: absolute;
+    bottom: 50px;
+    border-radius: 0px 0px 70px 70px;
+  }
+  .prong {
+    width: 20px;
+    height: 130px;
+    background: #060f55;
+    position: absolute;
+    bottom: 120px;
+    left: 130px;
+    border-radius: 70px;
+    box-shadow: 40px 0px #060f55, 80px 0px #060f55, 120px 0px #060f55;
```

Comment:
Again, not sure what box-shadow is for

Reply:
oh the box shadow is such that it repeats itself without me creating additional elements

Reply:
ahh ok smart